init.lua

```diff
@@ -1,4 +1,5 @@
 local load_time_start = minetest.get_us_time()
+math.randomseed(os.time())
```

Comment:
You don't need this.

Reply:
I think it's needed. Not using it made the pitch variation predictable. You can probably hear it too if you test it.

Reply:
The function is executed in builtin: https://github.com/minetest/minetest/blob/0c319469439237ba0db31ccbffeef06e2f5ab45f/builtin/init.lua#L23

Reply:
Hmm, need to look at it again to make sure.

Reply:
@tacotexmex